libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -3958,11 +3988,15 @@ instance (BaseUniverse fn, Member (FunFn fn) fn) => Functions (FunFn fn) fn wher
 lessOrEqualFn :: forall fn a. (Ord a, OrdLike fn a) => fn '[a, a] Bool
 lessOrEqualFn = injectFn (LessOrEqual @_ @fn)
 
+greaterOrEqualFn :: forall fn a. (Ord a, OrdLike fn a) => fn '[a, a] Bool
+greaterOrEqualFn = injectFn (GreaterOrEqual @_ @fn)
+
```

Comment:
Not necessary any more with the introduction of `flip_`. Let's remove it.

Reply:
follow on PR